pulp_container/tests/functional/api/test_recursive_add.py

```diff
@@ -76,17 +76,11 @@ def test_missing_repository_argument(self):
 
     def test_empty_source_repository(self):
         """Ensure exception is raised when source_repository does not have latest version."""
-        delete_response = self.repositories_api.delete(self.to_repo.latest_version_href)
-        monitor_task(delete_response.task)
         with self.assertRaises(ApiException) as context:
-            self.repositories_api.copy_manifests(
-                self.to_repo.pulp_href,
-                {
-                    # to_repo has no versions, use it as source
-                    "source_repository": self.to_repo.pulp_href,
-                }
-            )
+            self.repositories_api.delete(self.to_repo.latest_version_href)
```

Comment:
Where is the call to copy_manifests() ?

Reply:
I wasn't sure if I had to make this call, as for now pulpcore does not even start the task:
https://github.com/pulp/pulpcore/blob/master/pulpcore/app/viewsets/repository.py#L208-L215